src/client/components/common/Input/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { ElementType, FC, InputHTMLAttributes } from 'react';
+import React, { ElementType, FC, InputHTMLAttributes } from 'react';
```

Comment:
wrong import? @pentcle

Reply:
ah yes prob need to tell webstorm about tsx 
<img width="357" alt="image" src="https://user-images.githubusercontent.com/80323528/135619724-8050f2cb-1c5c-42ef-b7ee-867ad11179ed.png">

Reply:
vs code FTW